modules/final_radiological_review/php/NDB_Form_final_radiological_review.class.inc

```diff
@@ -190,7 +190,8 @@ class NDB_Form_final_radiological_review extends NDB_Form
                 LEFT JOIN tarchive t ON (upper(t.PatientName) 
                 LIKE CONCAT(upper(c.PSCID), '_', upper(s.CandID), '_', upper(s.visit_label), '%')) 
```

Comment:
I think you commented on the wrong line? (It's showing up as a comment on a line that doesn't contain any changes..)